lib/account/pages/account_page.dart

```diff
@@ -15,11 +20,68 @@ class _AccountPageState extends State<AccountPage> {
     supabaseClient.auth.signOut();
   }
 
+  void showDesignDialog(BuildContext context) {
+    showDialog(
+      context: context,
+      builder: (BuildContext context) => StatefulBuilder(
+        builder: (context, setState) => AlertDialog(
+          content: Column(
+            mainAxisSize: MainAxisSize.min,
+            children: List.generate(
+              ThemeMode.values.length,
+              (index) => RadioListTile(
+                  title: Text(
+                    ThemeMode.values[index].getName(context),
+                  ),
+                  value: ThemeMode.values[index],
+                  groupValue: themeManager.currentThemeMode,
+                  onChanged: (ThemeMode? value) {
+                    setState(() {
+                      themeManager.setTheme(value);
+                    });
+                  }),
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+
+  void changeTheme(ThemeMode? themeMode) {
```

Comment:
this method is never used i think. Can we remove it?

Reply:
Yes that's legacy 👍